### DISCUSSION TITLE
Do Firebase Cloud Functions work with ISR route rules?

### DISCUSSION THREAD

**Discussion by dosstx** [vuejs/vuefire#1506] [Questions and Help]
The Nuxt docs state that ISR only works with Netlify and Vercel, but the deployment example on Vuefire has an example with ISR. Is that correct and can be used with Firebase cloud functions ?

**Comment by posva** [comment]
I think the cached part might be the specific part that is only supported by vercel and netlify. You will have to ask firebase support to know if they have plans to support that
